lib/model/query/analytics.js

```diff
@@ -420,103 +420,116 @@ group by f."projectId"`);
+       , COALESCE(entz.num_entities_updated_total , 0) AS num_entities_updated_total
+       , COALESCE(entz.num_entities_updated_recent, 0) AS num_entities_updated_recent
+       , COALESCE(num_creation_forms              , 0) AS num_creation_forms
+       , COALESCE(num_followup_forms              , 0) AS num_followup_forms
+       , COALESCE(errors.total                    , 0) AS num_failed_entities_total
+       , COALESCE(errors.recent                   , 0) AS num_failed_entities_recent
+       , COALESCE(updates.total                   , 0) AS num_entity_updates_total
+       , COALESCE(updates.recent                  , 0) AS num_entity_updates_recent
+       , COALESCE(updates.update_sub_total        , 0) AS num_entity_updates_sub_total
+       , COALESCE(updates.update_sub_recent       , 0) AS num_entity_updates_sub_recent
+       , COALESCE(updates.update_api_total        , 0) AS num_entity_updates_api_total
+       , COALESCE(updates.update_api_recent       , 0) AS num_entity_updates_api_recent
+       , COALESCE(conflict_stats.conflicts        , 0) AS num_entity_conflicts
+       , COALESCE(conflict_stats.resolved         , 0) AS num_entity_conflicts_resolved
+       , COALESCE(creates.create_sub_total        , 0) AS num_entity_create_sub_total
+       , COALESCE(creates.create_sub_recent       , 0) AS num_entity_create_sub_recent
+       , COALESCE(creates.create_api_total        , 0) AS num_entity_create_api_total
+       , COALESCE(creates.create_api_recent       , 0) AS num_entity_create_api_recent
+       , COALESCE(bulk_creates.total              , 0) AS num_entity_create_bulk_total
+       , COALESCE(bulk_creates.recent             , 0) AS num_entity_create_bulk_recent
+  FROM datasets ds
+    LEFT JOIN (
+      SELECT "datasetId"
+           , COUNT(id) AS num_entities_total
+           , COUNT(CASE WHEN "createdAt" >= current_date - 45 THEN 1 END) as num_entities_recent
+           , COUNT(CASE WHEN "updatedAt" IS NOT NULL          THEN 1 END) AS num_entities_updated_total
+           , COUNT(CASE WHEN "updatedAt" >= current_date - 45 THEN 1 END) AS num_entities_updated_recent
+        FROM entities
+        GROUP BY "datasetId"
+    ) AS entz ON entz."datasetId" = ds.id
+    LEFT JOIN (
+      SELECT dfd."datasetId"
+           , COUNT(fd."formId") AS num_creation_forms
+        FROM dataset_form_defs AS dfd
+          JOIN form_defs AS fd ON fd.id = dfd."formDefId"
+        GROUP BY dfd."datasetId"
+    ) AS fd ON fd."datasetId" = ds.id
+    LEFT JOIN (
+      SELECT fa."datasetId"
+           , COUNT(fa."formId") AS num_followup_forms
+        FROM form_attachments AS fa
+          JOIN forms AS f ON f.id = fa."formId"
+        WHERE f."currentDefId" IS NOT NULL
+        GROUP BY fa."datasetId"
+    ) AS fa ON fa."datasetId" = ds.id
+    LEFT JOIN (
+      SELECT
+        COUNT (a.details -> 'submissionId'::TEXT) total,
+        SUM (CASE WHEN a."loggedAt" >= current_date - 45 THEN 1 ELSE 0 END) recent,
```

Comment:
Keep the 45 as a variable (even though we never change it / don't have a config to change it) or use `_cutoffDate`